Doc/library/pdb.rst

```diff
@@ -263,6 +263,19 @@ the commands; the input is split at the first ``;;`` pair, even if it is in the
 middle of a quoted string. A workaround for strings with double semicolons
 is to use implicit string concatenation ``';'';'`` or ``";"";"``.
 
+To set temporary global variables, use *convenience variable*. A *convenience
+variable* is a variable whose name starts with ``$``.  For example, ``$foo = 1``
+set a global variable ``$foo`` which you can use in the debugger session.  The
+*convenience variables* are cleared when the program resumes execution so it's
+less likely to interfere with your program compared to using normal variables
```

Comment:
is it likely at all, given its name is not a legal identifier?

Reply:
But python variables can't start with $.

Reply:
When a user does `$foo == 1`, it's equivalent to do `__pdb_convenience_variables['foo'] == 1`. If the user's code already has a variable named `__pdb_convenience_variables`, which is highly unlikely but not impossible, it will interfere with their existing code.